app/Notifications/AccountSuspended.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Individual;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Messages\VonageMessage;
+
+class AccountSuspended extends PlatformNotification
+{
+    public mixed $account;
+
+    public function __construct(mixed $account)
+    {
+        $this->account = $account;
+    }
+
+    public function toMail(): MailMessage
+    {
+        return (new MailMessage)
+            ->subject(__('Account suspended'))
+            ->line(__('Your account has been suspended.').' '.$this->getCapabilities($this->account).' '.__('Please contact us if you need further assistance.'))
+            ->action(__('Contact support'), localized_route('dashboard').'#contact');
+    }
+
+    public function toVonage(): VonageMessage
+    {
+        return (new VonageMessage)
+            ->content(
+                __('Your account on the Accessibility Exchange has been suspended.').' '.$this->getCapabilities($this->account).' '.__('Please contact us at :email or :phone if you need further assistance.',
+                    [
+                        'email' => settings()->get('email', 'support@accessibilityexchange.ca'),
+                        'phone' => phone(settings()->get('phone', '+1-888-867-0053'), 'CA')->formatForCountry('CA'),
+                    ]
+                )
+            )
+            ->unicode();
+    }
+
+    public function toArray(): array
+    {
+        return [
+            'title' => __('Your account has been suspended'),
+            'body' => __('Your account has been suspended.').' '.$this->getCapabilities($this->account).' '.__('Please contact us if you need further assistance.'),
+        ];
+    }
+
+    public function getCapabilities($notifiable): string
+    {
+        if ($notifiable instanceof Individual && ! $notifiable->isConnector() && ! $notifiable->isConsultant()) {
+            return __('You will not be able to edit any information in your account.');
+        }
+
+        return __('You will not be able to edit any information in your account. If you published your page already, it has been unpublished.');
```

Comment:
I guess this is true, but now I'm wondering if it should actually be unpublished (that is change the published state in the model). However, that would require them to re-publish the page when they are unsuspended.

Assuming we keep things as is, what would the expectation be for the admin when viewing a suspended user's page. It isn't published but it also isn't draft; which may have an effect on what we do for #1232

Reply:
@cherylhjli Can we adjust this copy to reflect the fact that pages remain published but are simply hidden from other users— something like "Your page will no longer be shown to other users of this website."

Reply:
that sounds good @greatislander